### PR TITLE
Use config var to turn force_ssl setting on or off.

### PR DESCRIPTION
**Pull request by monfresh**
Use the same setup as ohana-api, and clean up application.example.yml.

Use https in documentation that refers to demo sites.

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -134,7 +134,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = (ENV['ENABLE_HTTPS'] == 'yes')
```
Comment (monfresh) [inline config/environments/production.rb]: We're actually following the single quote style. I'll submit another PR with the proper Hound config.

**Comment by monfresh** [comment]
@anselmbradford Ready for review. Thanks!

File: script/setup_heroku [hunk 2]
```diff
@@ -64,4 +64,4 @@ echo "Pushing code to Heroku now. This will take a few minutes..."
 git push heroku master
 
 echo "All done pushing code."
-echo "You can now visit your site at http://$APP_NAME.herokuapp.com"
\ No newline at end of file
+echo "You can now visit your site at https://$APP_NAME.herokuapp.com"
```
Comment (anselmbradford) [inline script/setup_heroku]: Do you want a ending newline here?

File: config/application.example.yml [hunk 3]
```diff
@@ -33,13 +54,38 @@ OHANA_API_ENDPOINT: https://ohana-api-demo.herokuapp.com/api
...
 
+########################################
+#
+# ENABLE_HTTPS - REQUIRED IN PRODUCTION
+#
+########################################
+# When using the default Heroku domain setup (http://app-name.herokuapp.com),
+# you can take advantage of SSL for free. However, to use SSL with a custom
+# domain name, you'll need to do some work first, as explained in the Wiki:
+# https://github.com/codeforamerica/ohana-web-search/wiki/How-to-deploy-Ohana-Web-Search-to-your-Heroku-account#using-ssl-with-a-custom-domain
```
Comment (anselmbradford) [inline config/application.example.yml]: This URL is awful long, you might use a git.io URL instead: http://git.io/vez1W

**Comment by monfresh** [comment]
Thanks! I pushed the changes.

**Comment by anselmbradford** [comment]
:+1:

**Pull request merged**